### PR TITLE
Adding conflicts with pagerfanta 2

### PR DESCRIPTION
**Pull request by loic425**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -46,7 +46,12 @@
         "sylius/grid": "self.version"
     },
     "conflict": {
-        "twig/twig": "^1.0"
+        "twig/twig": "^1.0",
+        "pagerfanta/core": "^2.0",
+        "pagerfanta/doctrine-phpcr-odm-adapter": "^2.0",
+        "pagerfanta/doctrine-dbal-adapter": "^2.0",
+        "pagerfanta/doctrine-orm-adapter": "^2.0",
+        "pagerfanta/pagefanta": "^2.0"
```
Comment (mbabker) [inline composer.json:54]:
```suggestion
        "pagerfanta/core": "<3.0",
        "pagerfanta/doctrine-phpcr-odm-adapter": "<3.0",
        "pagerfanta/doctrine-dbal-adapter": "<3.0",
        "pagerfanta/doctrine-orm-adapter": "<3.0",
        "pagerfanta/pagefanta": "<3.0"
```

Conflicting only on `^2.0` would still allow 1.0 to be installed.

File: composer.json [hunk 2]
```diff
@@ -46,7 +46,12 @@
         "sylius/grid": "self.version"
     },
     "conflict": {
-        "twig/twig": "^1.0"
+        "twig/twig": "^1.0",
```
Comment (mbabker) [inline composer.json:49]:
```suggestion
        "twig/twig": "<2.12",
```

2.12 is the lowest tested version, I'm assuming that means it's also the lowest supported version and 2.0 thru 2.11 probably shouldn't be allowed either.

**Review by mbabker — APPROVED** [review]

**Review by Roshyo — APPROVED** [review]

**Review by Zales0123 — APPROVED** [review]

**Comment by Zales0123** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "fix/conflict-pagerfanta-2"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by Zales0123** [comment]
Thanks, Loïc! :tada:

**Pull request merged**